src/test/testKit/addCommentDriver.ts

```diff
@@ -0,0 +1,26 @@
+import type { Locator, Page } from 'playwright';
+import { COMMENTS, COMMENT_INPUT, POST_COMMENT_BUTTON } from './selectors';
+
+export class AddCommentDriver {
+    page: Page;
+    commentInputLocator: Locator;
+    postCommentButtonLocator: Locator;
+    commentsLocator: Locator;
+    constructor(page: Page) {
+        this.page = page;
+        this.commentInputLocator = page.locator(COMMENT_INPUT);
```

Comment:
No need to use "locator" in the variable name.

```
 clickPostCommentButton() {
        return this.postCommentButton.click();
    }
```

the code is more explicit this. way